src/components/ColumnTitle/ColumnTitle.tsx

```diff
@@ -24,12 +25,22 @@ export const ColumnTitle: FunctionComponent<ColumnTitle> = (props) => {
         return <div>{title}</div>
     }
 
-    const filterable = props.rowMeta.filterable
-    const sort = <ColumnSort
+    const noSortable = props.widgetMeta.type === FieldType.multivalue
+        || props.widgetMeta.type === FieldType.multifield
+        || props.widgetMeta.type === FieldType.hidden
+        || props.widgetMeta.type === FieldType.fileUpload
+        || props.widgetMeta.type === FieldType.checkbox
+        || props.widgetMeta.type === FieldType.pickList
```

Comment:
I think Picklists also should be sortable. They are major part of application, so if referencing some user with picklist then you'll definetly will want to sort them by they displayed value.  
We do have mapping between picklist value as id and as a displayed string so it should be possible, perhaps with a bit of additional logic, to allow sorting picklists

Reply:
Yes, that's right. Fixed.